evap/results/tools.py

```diff
@@ -51,6 +61,37 @@ def mix(a, b, alpha):
     return alpha * a + (1 - alpha) * b
 
 
+def mix_distributions(a, b, alpha):
+    sum_a = sum(a.values())
+    sum_b = sum(b.values())
+    distribution = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0}
+    if sum_a == 0 and sum_b == 0:
+        return distribution
+    if sum_a == 0:
+        return b
+    if sum_b == 0:
+        return a
```

Comment:
well, if we make sure the distribution adds up to 1 before returning it anywhere, the special cases are not important anymore.

Reply:
turns out we need the checks because a contributor might not have grade questions or non grade questions and then the other distribution is returned as is.